src/containers/ChapterContainer/route.js

```diff
@@ -0,0 +1,18 @@
+import loadable from 'loadable-components';
+
+export default [
+  {
+    path: '/:chapterId(\\d+)/:range/:translations',
+    component: loadable(
+      () => import(/* webpackChunkName: "ChapterContainer" */ './index') // import('./components/GlobalNav/Surah')
```

Comment:
**minor issue**

is there a better way to do these? seems like leaking web pack details is a smell.

Reply:
@sabeurthabti sorry, do you mean leaking the chunk name?

Reply:
It's not so bad to leak the chunk name. We do this commonly at Airbnb:

![image](https://user-images.githubusercontent.com/3749095/30721881-fb0cbc62-9ee2-11e7-9649-dfe053dfa471.png)

Helps us understand what chunk went wrong. That being said, we can change the webpack/prod.js file to output hashes

Reply:
I was mearly thinking from how easy we can make it for people who might not have been exposed to the concept of chunk names.

Reply:
agreed, it isn't so bad.

Reply:
Apologies, I may not understand :(